.github/workflows/integration-tests-non-root.yaml

```diff
@@ -70,6 +70,48 @@ jobs:
           chown -Rf ci:ci ${GITHUB_WORKSPACE} $(go env GOMODCACHE) $(go env GOCACHE)
         continue-on-error: true
 
+      - name: Install Docker CLI
```

Comment:
Can we add this to our buildbox instead of installing it on every run?

Reply:
sure